include/functions.h

```diff
@@ -616,7 +616,7 @@ PosRot* Actor_GetFocus(PosRot* dest, Actor* actor);
 PosRot* Actor_GetWorld(PosRot* dest, Actor* actor);
 PosRot* Actor_GetWorldPosShapeRot(PosRot* dest, Actor* actor);
 
-s32 Target_NotInLeashRange(Actor* actor, Player* player, s32 skipChecks);
+s32 Target_OutsideLeashRange(Actor* actor, Player* player, s32 ignoreLeash);
```

Comment:
Looking at this again though, how can you check if you're outside the leash range if you set `ignoreLeash` to true?

Reply:
:man_shrugging: 
That parameter kinda defeats the purpose of this function, it is weird

Reply:
You don't. By setting `ignoreLeash` you are automatically considered in range (it is used by player to give a few frames to stay targeted even the target moves outside the leash very fast).